presto-cassandra/src/main/java/io/prestosql/plugin/cassandra/CassandraPageSink.java

```diff
@@ -92,14 +95,14 @@ public CassandraPageSink(
             this.toCassandraDate = value -> LocalDate.fromDaysSinceEpoch(toIntExact(value));
         }
 
-        Insert insert = insertInto(schemaName, tableName);
+        Insert insert = insertInto(validSchemaName(schemaName), validTableName(tableName));
```

Comment:
This would belong to "Retain actual names in CassandraInsertTableHandle" if you extract it